src/worldcereal/utils/refdata.py

```diff
@@ -132,12 +136,116 @@ def query_public_extractions(
     public_df_raw = db.sql(main_query).df()
 
     # Process the parquet into the format we need for training
-    processed_public_df = process_parquet(public_df_raw)
+    processed_public_df = process_parquet(public_df_raw, processing_period)
 
     return processed_public_df
 
 
-def process_parquet(public_df_raw: pd.DataFrame) -> pd.DataFrame:
+def month_diff(month1: int, month2: int) -> int:
```

Comment:
I may be overlooking something (though I tested quite a bit), but can the same not be accomplished with a very simple function instead?

```
def month_diff(month1: int, month2: int) -> int:
    return month2 - month1 if month2 >= month1 else 12 - month1 + month2
```

Another comment: it's documented "the number of months between two given months". If I'd say the number of months between January and March, what do you expect? There's one month between them, February, the total span is three months, Jan-Feb-Mar. So think carefully about what the functionality you actually need, and then make sure it's documented like this as well.